### PR TITLE
Refactor and improve error tracking

### PR DESCRIPTION
**Pull request by MasterCruelty**
- I changed the colour of crossmarks in <code>integrity_check.sh</code>
- I updated <b>printProgress</b> function to recognize another word .
- I added a furhter condition in <code>emerger.sh</code> to verify and tell to the user if <code>.errors</code> is containing something. Solving #60

### CODE REVIEW

**Review by MasterCruelty — COMMENTED** [review]
It works for me. @archetipico if you have any comment, otherwise I'll merge this changes to main branch.

**Review by MasterCruelty — COMMENTED** [review]
I Improved error tracking again by adding another argument <code>-err</code>  @archetipico

**Review by MasterCruelty — COMMENTED** [review]
Added the case of empty <code>.errors</code> for <code>-err</code>

**Comment by archetipico** [comment]
I can't test it these days, let's accept it and if it goes bad we roll back

**Comment by archetipico** [comment]
I read the changes but I think the issue should still be opened. We didn't implement a way to clear `.errors` automatically.

File: src/utils/global.sh [hunk 1]
```diff
@@ -27,7 +27,7 @@ TRASH="\U0001F4A9"
 WHALE="\U0001F40B"
 
 printProgress() {
-    if [[ $1 == *starting ]] || [[ $1 == *privileges* ]] || [[ $1 == *already* ]] || [[ $1 == *aborting* ]]; then
+    if [[ $1 == *starting ]] || [[ $1 == *privileges* ]] || [[ $1 == *already* ]] || [[ $1 == *aborting* ]] || [[ $1 == *Warning* ]]; then
```
Comment (archetipico) [inline src/utils/global.sh:30]: We will have to change this

**Review by archetipico — COMMENTED** [review]
:ok_hand:

File: src/utils/global.sh (hunk 1 above)
Comment (MasterCruelty) [reply src/utils/global.sh:30]: > We will have to change this

Yea I know, it's ugly

**Pull request merged**